src/Tizen.NUI.Components/Controls/Navigation/ContentPage.cs

```diff
@@ -136,34 +135,41 @@ public View Content
                     return;
                 }
 
-                content.Weight = 1.0f;
+                Add(content);
 
-                ResetContent();
+                CalculatePosition();
             }
         }
 
-        private void ResetContent()
+        /// <inheritdoc/>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override void OnRelayout(Vector2 size, RelayoutContainer container)
         {
-            // To keep the order of AppBar and Content, the existing contents are
-            // removed and added again.
-            if ((appBar != null) && Children.Contains(appBar))
-            {
-                Remove(appBar);
-            }
+            base.OnRelayout(size, container);
 
-            if ((content != null) && Children.Contains(content))
-            {
-                Remove(content);
-            }
+            CalculatePosition();
+        }
 
-            if (appBar != null)
+        // Calculate appBar and content's positions.
+        private void CalculatePosition()
+        {
+            if (appBar == null)
             {
-                Add(appBar);
+                if (content != null)
+                {
+                    content.Position2D = new Position2D(Position2D.X, Position2D.Y);
+                    content.Size2D = new Size2D(Size2D.Width, Size2D.Height);
+                }
             }
-
-            if (content != null)
+            else
             {
-                Add(content);
+                appBar.Position2D = new Position2D(Position2D.X, Position2D.Y);
```

Comment:
also can be applied Margin and Paddings.

Reply:
Thank you for your comment!
The code has been updated.